### ISSUE TITLE
surface the transaction hash for a given proposal in app.uniswap.org/#/vote 

### ISSUE THREAD

**Issue by Yorkemartin** [Uniswap/interface#3469]
app.uniswap.org/#/vote surfaces the wallet from which a proposal was submitted, but not the tx hash itself. Finding a tx hash of a given proposal becomes oddly difficult as a result. 

for example, we have 

```
Proposer
[0x9B68c14e936104e9a7a24c712BEecdc220002984](https://etherscan.io/address/0x9B68c14e936104e9a7a24c712BEecdc220002984)
```

I suggest we amend to 

```
Proposer
[0x9B68c14e936104e9a7a24c712BEecdc220002984](https://etherscan.io/address/0x9B68c14e936104e9a7a24c712BEecdc220002984)

Transaction 
[TX_ HASH_HERE](ETHERSCAN_URL_HERE)
```
Seems like a small lift if we already have the proposal sender

**Comment by uniyj** [comment]
Yes please! This was also raised on Uniswap Discord by a user who is looking to meet the Unigrants Community Analytics [bounty on governance](https://unigrants.notion.site/Bounty-4-Uniswap-Governance-f4d4936c09e440b8ab173a5a2804e663), and found that it was difficult to get the proposal tx, exactly as you described above.
